docs/tutorial/building-a-theme.md

```diff
@@ -0,0 +1,1548 @@
+import merge from "lodash.merge"
+import { theme } from "@jlengstorf/gatsby-theme-events"
+
+export default merge({}, theme, {
+  colors: {
+    primary: "blue",
+  },
+})
+```
+
+You'll be using `lodash.merge`, so install that now:
+
+```shell
+yarn add lodash.merge
+```
+
+Restart the dev server for `theme-test`. Your local site should now have a blue header instead of a purple one:
+
+![Test site, with banner color overridden.](./images/building-a-theme-override-colors.png)
+
+A few notable things are happening in this `index.js` file:
+
+- The `theme` import from `@jlengstorf/gatsby-theme-events` is the base UI theme from `@jlengstorf/gatsby-theme-events`.
+- The new object exported from `index.js` uses `lodash.merge` to deeply merge the base UI theme with the theme overrides of your choice. In this case, changing the primary color to blue.
+
+### Override an entire component
+
+Inside `src`, create a folder with the same title as your theme.
+
+> 💡 Note, with a namespaced theme, this will be two directories deep. For example, using the example of `@jlengstorf/gatsby-theme-events`, the file tree of your project will look like this:
+
+```text
+.
+├── data
+│   └── events.yml
+├── src
+│   ├── @jlengstorf
+│   │   └── gatsby-theme-events
+│   └── gatsby-plugin-theme-ui
+│       └── index.js
+├── .gitignore
+├── gatsby-config.js
+├── package.json
+└── yarn.lock
+```
+
+Anything inside `theme-test/src/@jlengstorf/gatsby-theme-events` will "shadow" the components in `@jlengstorf/gatsby-theme-events`.
+
+For example, create a new file to override the layout component: `theme-test/src/@jlengstorf/gatsby-theme-events/components/layout.js`.
+
+```javascript:title=theme-test/src/@jlengstorf/gatsby-theme-events/components/layout.js
+import React from "react"
+
+export default ({ children }) => <>{children}</>
+```
+
+If you restart the development server, you'll see all of the styles and structure from the theme have been stripped away, because the component has been completely overridden:
+
+![Test site, with layout component overridden.](./images/building-a-theme-override-component.png)
+
+## Conclusion
+
+Congratulations! You've set up, built, and customized your first Gatsby theme!
+
+### Further reading
+
+- Check out the [Gatsby themes docs](/docs/themes/) to keep learning.
+- Have an idea for a theme you'd like to build? Get a headstart on a local theme development workspace using the [Theme Workspace starter](https://github.com/gatsbyjs/gatsby/tree/master/themes/gatsby-starter-theme-workspace).
+- Have you built a Gatsby starter before? Perhaps [convert that starter to a theme](/docs/themes/converting-a-starter).
+- All through the month of July 2019, Jason Lengstorf will be live-streaming building themes with community members. Follow his Twitch channel for updates, or check out recordings from previous streams:
+  - [Building your first Gatsby theme (with Emma Wedekind)](https://youtu.be/W2uTfay3doo)
```

Comment:
```suggestion
```

Reply:
This is going to be hard to keep up-to-date (there are already two more streams to add to it), so a link to the playlist will be easier to manage I think.